packages/showcases/css/README.md

```diff
@@ -2,34 +2,12 @@
 
 > Decathlon Design System - Vitamin CSS showcase
 
-## Technologies
+<a href="https://decathlon.github.io/vitamin-web/@vtmn/showcase-css"><img src="https://img.shields.io/badge/storybook-css-d891bc?style=flat&logo=storybook" alt="storybook" /></a>
 
-- [Storybook](https://storybook.js.org/) - CSS guidelines & components are exposed in a Storybook to help you understand how to consume this package.
+## I have an issue, what should I do?
 
-## Contributing
-
-One of the Decathlon Design System goals is to provide guidelines & components to gain in consistency, efficiency & accessibility. The best way to achieve this is together!
-That's why we are on GitHub. We would love contributions from the community (bug reports, feature requests, suggestions, Pull Requests, whatever you want!).
-
-### Installation
-
-In this monorepo, we use [Yarn](https://yarnpkg.com) & [Lerna](https://github.com/lerna/lerna).
-Read the [Installation Guide](https://yarnpkg.com/en/docs/install) for detailed instructions on how to install Yarn.
-
-Once you have installed [Yarn](https://yarnpkg.com), please run:
+Please file the issue [here](https://github.com/Decathlon/vitamin-web/issues/new).
 
-```sh
-yarn
-```
-
-> 👉 See [the contributing docs](CONTRIBUTING.md) for more information about how to contribute.
-
-### Development
-
-To develop in watch mode and run all showcases with hot reloads, just run:
-
-```sh
-yarn start
-```
+## Contributing
 
-Then, the showcase on [localhost:6006](http://localhost:6006).
+See [the contributing docs](https://github.com/Decathlon/vitamin-web/blob/main/CONTRIBUTING.md).
```

Comment:
It's no longer necessary to display the license in the "showcase" files?

Reply:
It's not the license but the contributing part :)